src/entry.c

```diff
@@ -677,7 +677,24 @@ get_range(bitstr_t * bits, int low, int high, const char *names[],
 	if (state != R_FINISH || ch == EOF)
 		return (EOF);
 
-	for (i = num1; i <= num2; i += num3)
+	/* Make sure the step size makes any sense */
+	if (step > 1) {
+		if (high == high_ && low == low_) {
+			if (step > (high_ - low_)) {
+				fprintf(stderr, "Step size of '%i' will be ignored, since it exceeds the maximum possible step size of '%i' for the specified entry'\n", step, high_ - low_);
```

Comment:
Could you please make the messages more terse so they do not exceed the 80 characters length?

I.e. `Warning: Step size %i higher than possible maximum of %i`

Reply:
ok, done.